### ISSUE TITLE
Active Assignment using AADRoleEligibilitySchedulRequest - Help Please

### ISSUE THREAD

**Issue by tayhall** [microsoft/Microsoft365DSC#5021]
I am trying to assign roles to a group and have the role active. I can only seem to use the RoleEligibilityScheduleRequest to assign the role as Eligible to the group.

Am I trying to do something that's not possible or what should I be setting to get an Active Eligibility.

I was using AdminAssign but cannot find any fields for Active versus Eligible.

Ive managed to find referecnes for the role assignment request. The possible values are: adminAssign, adminUpdate, adminRemove, selfActivate, selfDeactivate, adminExtend, adminRenew, selfExtend, selfRenew, unknownFutureValue.
adminAssign: For administrators to assign roles to principals.
adminRemove: For administrators to remove principals from roles.
adminUpdate: For administrators to change existing role assignments.
adminExtend: For administrators to extend expiring assignments.
adminRenew: For administrators to renew expired assignments.
selfActivate: For principals to activate their assignments.
selfDeactivate: For principals to deactivate their active assignments.
selfExtend: For principals to request to extend their expiring assignments.
selfRenew: For principals to request to renew their expired assignments.

**Comment by FabienTschanz** [comment]
@tayhall Do you want to assign an entra role to a PIM group so that it's always active? If yes, then you can use the `AssignedToRole` property on `AADGroup` to assign it permanently.

**Comment by tayhall** [comment]
Yes that was the plan, I ended up assigning active roles via AADGroups and
then manage membership via PIM.
